Packs/MicrosoftCloudAppSecurity/Integrations/MicrosoftCloudAppSecurity/MicrosoftCloudAppSecurity.yml

```diff
@@ -61,8 +61,9 @@ configuration:
   type: 16
 - additionalinfo: 'A custom filter by which to filter the returned files. If you pass
     the custom_filter argument it will override the other filters from the integration
-    instance configuration. Example for Custom Filter: {"entity.policy":{"eq":"Impossible
-    travel"}}. For more information about filter syntax, please refer to https://docs.microsoft.com/en-us/cloud-app-security/api-alerts#filters.'
+    instance configuration. Example for Custom Filter: {"severity":{"eq":2}}.
+    Note that for filtering by "entity.policy", one should use the id of the policy.
```

Comment:
Is there a way to give an example for the use of `entity.policy` this seems a bit vague

Reply:
example was added